geoSegment/utils/config.js

```diff
@@ -0,0 +1,39 @@
+const s = require('@imin/speck');
+const fs = require('fs-extra');
+const path = require('path');
+const { log } = require('./log');
+
+const CONFIG_FILE_PATH = path.join(__dirname, '..', 'config.json');
+
+const Config = s.type({
+  firehoseBaseUrl: s.string,
+  firehoseApiKey: s.string,
+  segments: s.array(s.type({
+    identifier: s.string,
+    latitude: s.float,
+    longitude: s.float,
+    radius: s.float,
+    attendanceModeFilter: s.literalStringEnum(/** @type {['virtual-only', 'physical-only', 'all']} */([
```

Comment:
😭  in .ts files you can just use `as const`

Reply:
The reason I'm doing this here is that otherwise TS reads this as "an array of strings", which it then turns into the type `string` - but I want a union of these three exact string literals